programmers/문자열-내림차순으로-배치하기.js

```diff
@@ -0,0 +1,4 @@
+function solution(s) {
+  var answer = s.split('').sort().reverse().join('');
+  return answer;
```

Comment:
단순 구현일 경우 굳이 변수에 담을 필요는 없습니다.
변수를 담는 이유는 해당 값이 어떤 의미를 갖는지 알기 위함인데
이와 같은 경우는 함수명으로 값의 의미를 표현하면 됩니다. 
따라서 함수명만 잘 짓는다면 한줄로 반환해버려도 좋은 코드라 할 수 있지요.
```suggestion
  return s.split('').sort().reverse().join('');
```